subsys/pm/device.c

```diff
@@ -24,12 +24,26 @@ const char *pm_device_state_str(enum pm_device_state state)
 	}
 }
 
+static int domain_supported_cb(const struct device *dev, void *context)
+{
+	struct pm_device *pm = dev->pm;
+	enum pm_device_action action = (enum pm_device_action)context;
+
+	if (pm->action_cb != NULL) {
+		(void)pm->action_cb(dev, action);
```

Comment:
are all errors ignored on purpose? maybe just `-ENOTSUP` should be ignored?

Reply:
I think ignoring errors here is a dangerous practice: system will be left in a state where certain devices will not be operational without any notice.